db-scripts/createdb.sql

```diff
@@ -71,7 +73,8 @@ create table if not exists equipment_rentals(
    user_id text,
    date_borrowed date not null default current_date,
    return_date date not null,
-   price money not null,
+--   price money not null,
+   price numeric(10,2) not null,
```

Comment:
`money` type is not compatible with drizzle and it seemed to be more limited, we may also need to update our ER diagram for latest reference

Reply:
what sort of updates